ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template4.kt

```diff
@@ -274,10 +274,20 @@ private fun SelectPackageButton(
     ) {
         DiscountRelativeToMostExpensivePerMonth(colors = colors, text = discountText, selected = isSelected)
 
-        Box {
+        Box(
+            modifier = Modifier
+                // Apply the border to the box instead of the button to avoid the border showing
+                // a white line around
+                .border(
+                    width = UIConstant.defaultPackageBorderWidth,
+                    color = mainColor,
+                    shape = RoundedCornerShape(UIConstant.defaultCornerRadius),
+                ),
+        ) {
             Button(
                 modifier = modifier
                     .alpha(buttonAlpha)
+                    .padding(1.dp) // Trick to prevent white line around the button border
```

Comment:
I wonder if it would work with 1px instead of 1dp... You can convert pixels to dp with this I think: https://developer.android.com/reference/kotlin/androidx/compose/ui/unit/Density#(kotlin.Int).toDp()

Reply:
it worked! perfect